inc/namespace.php

```diff
@@ -1978,6 +1978,8 @@ function handle_autosuggest_endpoint() {
 	$search = $features->get_registered_feature( 'search' );
 
 	// Force post filter value.
+	// This is necessary because the ES query is passed frmo the client side,
+	// and we want to ensure only published, searchable content.
```

Comment:
```suggestion
	// This is necessary because the ES query is passed from the client side,
	// and we want to ensure only published, searchable content is returned.
```